definitions/npm/@ginkgo-bioworks/react-json-schema-form-builder_v2.x.x/flow_v0.92.x-/react-json-schema-form-builder_v2.x.x.js

```diff
@@ -0,0 +1,110 @@
+declare module '@ginkgo-bioworks/react-json-schema-form-builder' {
+  declare type Parameters = {|
+    [string]: string | number | boolean | Array<string | number>,
+    name: string,
+    path: string,
+    definitionData: { [string]: any, ... },
+    definitionUi: { [string]: any, ... },
+    category: string,
+    'ui:option': { [string]: any, ... },
+  |};
+
+  declare export type CardBodyProps = {|
+    parameters: Parameters,
+    onChange: (newParams: Parameters) => void,
+  |};
+
+  declare type DataType =
+    | 'string'
+    | 'number'
+    | 'boolean'
+    | 'integer'
+    | 'array'
+    | '*'
```

Comment:
Maybe remove this if possible, do you mean to use `void` or something? https://flow.org/en/docs/types/utilities/#toc-existential-type

Reply:
Removed. (Also removed the null, which is incorrect.)